programs/autotension.cpp

```diff
@@ -245,8 +292,13 @@ void AutoTension<DOF>::init(ProductManager& pm, std::vector<int> args) {
 	jpSlack1.resize(DOF);
 	jpSlack2.resize(DOF);
 
+	// Commenting the following block of code: the stack is 
+	// going out of real-time control when trying to initialize hand.
+	// TODO (avk): Find out what the issue is.
```

Comment:
Does this issue still happen with @egordon's patches on top of the original PR?

Reply:
@aditya-vk We should test this.

Reply:
Right. I can test this next weekend, unless you can do it before. In any case, it is important to note that the hand should be closed before the tensioning process begins.